.gitignore

```diff
@@ -0,0 +1,11 @@
+menu.svg
+arc.svg
+bar.svg
+bird.svg
+c.svg
+cat.svg
+circle.svg
+m.svg
+MyLogo.svg
+n.png
+two.svg
```

Comment:
- [ ] [REQUIRED] Please, remove the images from the `.gitignore` file. and add the `node_modules/` instead.

Reply:
Good job in implementing the changes 👍